unturned/OpenMod.Unturned/Vehicles/Events/UnturnedVehicleLockedEvent.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenMod.Unturned.Players;
+using Steamworks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenMod.Unturned.Vehicles.Events
+{
+    public class UnturnedVehicleLockedEvent : UnturnedVehicleEvent
```

Comment:
Add locked state property. This is available directly from the vehicle instance, but relevant info should be contained within the event's class.